test/fixtures/ecpair.json

```diff
@@ -82,7 +82,27 @@
       {
         "exception": "Private key must be less than the curve order",
         "d": "115792089237316195423570985008687907853269984665640564039457584007913129639935"
+      },
+      {
+        "exception": "Expected property \"compressed\" of type \\?Boolean, got Number 2",
```

Comment:
Because this was previously not under test,  turns out the `Error` message was completely wrong.
This is what was being emitted.